### PR TITLE
pass aspectRatio to React Native

### PR DESCRIPTION
**Pull request by piranha**
There is an issue #370 about just that, and I needed it, so here we go. 😁

### CODE REVIEW

**Comment by adamstep** [comment]
Thanks for the contribution! Looks like the failing test is related for code formatting, if you run `yarn prettify` that should take care of it.

**Comment by piranha** [comment]
Oh right! Updated with prettify's changes now! Tests are still broken though... :(

**Comment by adamstep** [comment]
Looks like something related to the XML schema changes. We validate the schema against all of the XML files in https://github.com/Instawork/hyperview/tree/master/examples. Let me take a look...

File: schema/core.xsd [hunk 1]
```diff
@@ -470,6 +485,8 @@
         </xs:simpleType>
       </xs:attribute>
 
+      <xs:attribute name="aspectRatio" type="hs:float" />
```
[the remark this answers is not in the document]
Comment (piranha) [reply schema/core.xsd:488]: 🤦‍♂️ I'm sorry! I have this change locally along with a few other, and didn't update the commit somehow. :\

**Review by adamstep — APPROVED** [review]
looks good, thank you!

**Pull request merged**